nh_graphs/dev/coffee/tests/lib/helpers.js

```diff
@@ -134,3 +134,8 @@ if (!Function.prototype.bind) {
   };
 }
 
+function skipIfCantResize(){
+    if(!!window.chrome && !!window.chrome.webstore){
+       pending();
+    }
+}
```

Comment:
Add a tech debt to fix the underlying problem of the test helper function not being able to resize the window due to Chrome's lack of support for that event.